### PR TITLE
#1531 fixed typos and improved some of the grammar in quiz level 1

### PR DESCRIPTION
**Pull request by johannadekker**
I fixed both typos in questions 7 and 9. I also noticed some of the feedback had some very small errors that I fixed. For example:

- In level 1 question 5, one of the options was echo misspelled as ego. However, this only works as a spelling mistake in Dutch. I changed it to 'ekko' so that it would remain similar to the original Dutch version.
- In level 1 question 4, feedback for option B first said: ""Im is spelled wrong on purpose**,** Hedy doesn't understand the apostrophes yet." However, that is incorrect grammar as these could be two separate sentences. I changed the comma to a semi-column ( ; ).
- One of the options for question 1, what language is this was the name 'Henk'. For an anglophone child, that is an unfamiliar name. I changed it to Heidi, which should be more familiar.
- The feedback for the 'no' option in Q10, asking if they want to progress to the next level was changed from 'Too bad' to 'What a shame'. I believe this is a more accurate translation of 'Jammer'. I think too bad could be experienced as sarcastic.
- Somewhere, in the feedback for a wrong answer, it said correc instead of correct. I fixed that too.

tl;dr : Fixed small language errors

### CODE REVIEW

**Comment by Felienne** [comment]
Thanks for the fixes @johannadekker but this PR is a bit far behind main to be merged (since we now closed #777) I will make a new PR with these changes in it.
